README.md

```diff
@@ -139,6 +139,19 @@ See : https://sql-js.github.io/sql.js/examples/GUI/gui.js
 
 #### Loading a database from a server
 
+##### using fetch
+
+```javascript
+  const SQL = await initSqlJs({
+    locateFile: file => `https://path/to/your/dist/folder/dist/${file}`
+  });
+  const dbres = await fetch("/path/to/databse.sqlite");
```

Comment:
You don't need to wait for sql.js to have loaded to start making the request to your database. I would use `Promise.all`, or put the `await`s after the two promise creations.